### ISSUE TITLE
Remove ServiceStack.Text library and replace with Json.net

### ISSUE THREAD

**Issue by aaronmell** [KyleGobel/Hipchat-CS#32]
The license is restrictive, and requires purchase if used for commercial purposes. This makes it difficult for a business to use.

**Comment by KivraS** [comment]
I agree , i did not install the library just because of this

**Comment by KyleGobel** [comment]
I agree, it should be swapped out with a different JSON library (json.net
being probably the best option).  If anyone wants to start a new
fork/brank/whatever and work on this that'd be great.  The company I worked
for no longer used/uses hipchat, so my ability to work on this is limited.

**Comment by kilroytrout** [comment]
Latest release of ServiceStack.Text is now free for commercial use.
